app/controllers/facility_statements_controller.rb

```diff
@@ -67,11 +67,13 @@ def send_statements
         flash[:error] = I18n.t("controllers.facility_statements.errors_html", errors: @errors.join("<br/>")).html_safe
         raise ActiveRecord::Rollback
       else
-        @account_statements.each do |account, statement|
-          account.notify_users.each { |u| Notifier.delay.statement(user: u, facility: statement.facility, account: account, statement: statement) }
+        unless SettingsHelper.feature_on?(:skip_statement_emails)
+          @account_statements.each do |account, statement|
+            account.notify_users.each { |u| Notifier.delay.statement(user: u, facility: statement.facility, account: account, statement: statement) }
+          end
         end
         account_list = @account_statements.map { |a, _s| a.account_list_item }
-        flash[:notice] = I18n.t("controllers.facility_statements.send_statements.success_html", accounts: account_list.join("<br/>")).html_safe
+        flash[:notice] = I18n.t("controllers.facility_statements.create_statements.success_html", accounts: account_list.join("<br/>")).html_safe
```

Comment:
I wonder if it would be worth taking the time while we're in to switch this to use `text_helpers`.